src/QueueReceiver.Infrastructure/Repositories/PlantRepository.cs

```diff
@@ -17,14 +17,16 @@ public PlantRepository(QueueReceiverServiceContext context)
 
         public IEnumerable<string> GetAllInternalAndAffiliateOids()
```

Comment:
This method is actually not used any more. Seems like a good chance for some housekeeping, can you please delete this and `GetMemberOidsByPlant` which is also not used. Also the interface references + similar methods in `PlantService` :)

Reply:
Removed